src/databricks/labs/ucx/mounts/list_mounts.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from dataclasses import dataclass
+
+from databricks.sdk import WorkspaceClient
+
+# Added a * import otherwise it won't be possible to import spark directly
+from databricks.sdk.runtime import *  # noqa: F403
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class MountResult:
+    name: str
+    source: str
+    instance_profile: str | None = None
```

Comment:
```suggestion
```
remove, for now. mount + IAM profile association is probably better done from a separate table